roles/lib_utils/callback_plugins/aa_version_requirement.py

```diff
@@ -7,6 +7,8 @@
 first (alphanumerically) by Ansible.
 """
 import sys
+from distutils import version
```

Comment:
```
roles/lib_utils/callback_plugins/aa_version_requirement.py:10: [E0611(no-name-in-module), ] No name 'version' in module 'distutils'
```

Reply:
@vrutkovs Which python version and distro is this running on? Cause even on the minimal centos:7 container there's python-distutils and on fedora if you install python (both 2 and 3) then it works.

```
~ 🎩 sudo docker run -it --rm centos:7 /bin/bash
[sudo] password for celebdor: 
[root@cd6b9cf20fe9 /]# python
Python 2.7.5 (default, Jul 13 2018, 13:06:57) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-28)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from distutils import version
>>> 
[root@cd6b9cf20fe9 /]#  from distutils import version 
bash: from: command not found
[root@cd6b9cf20fe9 /]# python
Python 2.7.5 (default, Jul 13 2018, 13:06:57) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-28)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from distutils import version 
>>> 
[root@cd6b9cf20fe9 /]# exit
~ 🎩︎ sudo docker run -it --rm fedora:28 /bin/bash
[root@dcb0f2032b96 /]# python
bash: python: command not found
[root@dcb0f2032b96 /]# dnf install python2 python3
Fedora 28 - x86_64 - Updates                                                                                                                                                       6.8 MB/s |  26 MB     00:03    
Fedora 28 - x86_64                                                                                                                                                                 5.0 MB/s |  60 MB     00:12    
Last metadata expiration check: 0:00:07 ago on Thu Oct 25 10:00:41 2018.
Package python3-3.6.6-1.fc28.x86_64 is already installed, skipping.
Dependencies resolved.
===================================================================================================================================================================================================================
 Package                                                   Arch                                          Version                                              Repository                                      Size
===================================================================================================================================================================================================================
Installing:
 python2                                                   x86_64                                        2.7.15-4.fc28                                        updates                                        102 k
Installing dependencies:
 python2-libs                                              x86_64                                        2.7.15-4.fc28                                        updates                                        6.3 M
 python2-pip                                               noarch                                        9.0.3-2.fc28                                         updates                                        2.0 M
 python2-setuptools                                        noarch                                        39.2.0-6.fc28                                        updates                                        644 k

Transaction Summary
===================================================================================================================================================================================================================
Install  4 Packages

Total download size: 9.1 M
Installed size: 37 M
Is this ok [y/N]: y
Downloading Packages:
(1/4): python2-2.7.15-4.fc28.x86_64.rpm                                                                                                                                            229 kB/s | 102 kB     00:00    
(2/4): python2-setuptools-39.2.0-6.fc28.noarch.rpm                                                                                                                                 1.0 MB/s | 644 kB     00:00    
(3/4): python2-pip-9.0.3-2.fc28.noarch.rpm                                                                                                                                         1.3 MB/s | 2.0 MB     00:01    
(4/4): python2-libs-2.7.15-4.fc28.x86_64.rpm                                                                                                                                       3.1 MB/s | 6.3 MB     00:02    
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Total                                                                                                                                                                              2.9 MB/s | 9.1 MB     00:03     
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                                                                                                                           1/1 
  Installing       : python2-libs-2.7.15-4.fc28.x86_64                                                                                                                                                         1/4 
  Installing       : python2-pip-9.0.3-2.fc28.noarch                                                                                                                                                           2/4 
  Installing       : python2-setuptools-39.2.0-6.fc28.noarch                                                                                                                                                   3/4 
  Installing       : python2-2.7.15-4.fc28.x86_64                                                                                                                                                              4/4 
  Running scriptlet: python2-2.7.15-4.fc28.x86_64                                                                                                                                                              4/4 
[root@dcb0f2032b96 /]# python2 -c 'from distutils import version'
[root@dcb0f2032b96 /]# python3 -c 'from distutils import version'
[root@dcb0f2032b96 /]# exit
```

Reply:
I think this is the issue: https://github.com/PyCQA/pylint/issues/73